pkgs/games/anki/addons/anki-button-colours/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "anki-button-colours";
+  version = "2.1.1";
+  src = fetchFromGitHub {
+    owner = "teaqu";
+    repo = "anki-button-colours";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-PI2Whtm5/q/X4cnFyANjFQtErOXqdDTSBWPXJjlyTj8=";
+  };
+  passthru.updateScript = nix-update-script { };
+  meta = {
+    description = "Button colours for Anki";
```

Comment:
It would be nice to provide an example for changing the config like is done here: https://github.com/NixOS/nixpkgs/blob/fe160d408a55b77bf10830b39e5f4c5ea8fc6314/pkgs/games/anki/addons/local-audio-yomichan/default.nix#L38-L67

Especially since this add-on doesn't have UI for changing the config (which would end up giving the user what they need to put into their Nix code once the config write fails).